### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by Manfred1958**
Describe the changes you have conducted here

Preview the messages on https://fink.inlang.com/github.com/Manfred1958/soaringmeteo .

### CODE REVIEW

**Comment by Manfred1958** [comment]
First translation, 'test'

**Comment by Manfred1958** [comment]
First translation 'test'

**Comment by julienrf** [comment]
Thank you @Manfred1958! I will ask another member of the “Deutch team” to review the translations as soon as possible :wink:

File: frontend/messages/de.json [hunk 1]
```diff
@@ -2,9 +2,19 @@
 	"$schema": "https://inlang.com/schema/inlang-message-format",
 	"help": "Hilfe",
 	"menuDisplayOnMap": "Auf der Karte anzeigen",
+	"menuOverlay": "Überlagerung",
```
Comment (Boran) [inline frontend/messages/de.json:5]: Then also "Überlagerung auf der Karte anzeigen" for next line

Comment (Manfred1958) [reply frontend/messages/de.json:5]: In German everybody understands also 'Overlay'. If you want...

**Comment by Manfred1958** [comment]
Julien or Boran, I'm still not quite sure: Do I always have to press the 'close with comments' button after my translation session is done? Or do we always leave the 'update translations' open?

**Comment by Manfred1958** [comment]
Correction and further translation 29 2. 11:04

**Comment by Boran** [comment]
looks good

**Comment by julienrf** [comment]
Thank you @Manfred1958!

**Comment by julienrf** [comment]
> Julien or Boran, I'm still not quite sure: Do I always have to press the 'close with comments' button after my translation session is done? Or do we always leave the 'update translations' open?

That’s a good question! The translation system uses several tools together and it’s not easy to understand all the parts :)

Once you opened the “pull request” on GitHub you can still continue to translate new messages, they will be added to the open pull request. You should never “close with comment” a pull request, unless you want to cancel your contribution for some reason.

**Comment by julienrf** [comment]
@Manfred1958, FYI, before you start translating new messages you should synchronize [your fork](https://github.com/Manfred1958/soaringmeteo) with the [upstream repository](https://github.com/soaringmeteo/soaringmeteo) by clicking to “Sync fork” in GitHub. I’ve updated the [translation instructions](https://github.com/soaringmeteo/soaringmeteo/blob/main/TRANSLATING.md#fix-or-complete-an-existing-translation) (step 4) with some relevant screenshots in case that helps!

**Pull request merged**